cli/crates/cli/src/output/report.rs

```diff
@@ -149,6 +149,32 @@ pub fn udf_message(udf_kind: UdfKind, udf_name: &str, message: &str, level: UdfM
     }
 }
 
+pub fn operation_started(_request_id: &str, _name: &Option<String>) {}
+
+pub fn operation_completed(
+    _request_id: &str,
+    name: Option<String>,
+    r#type: common::types::OperationType,
+    duration: std::time::Duration,
+    debug: bool,
+) {
+    if let common::types::OperationType::Query { is_introspection: true } = r#type {
+        if !debug {
+            return;
+        }
+    }
+
+    let formatted_duration = if duration < std::time::Duration::from_secs(1) {
+        format!("{:.2}ms", duration.as_secs_f64() / 1000.0)
+    } else {
+        format!("{:.2}s", duration.as_secs_f64())
+    };
```

Comment:
Noticed that introspection in your screenshots said `0ms` - do any of our actual queries return that fast?  Might be nice to do a `ns` option as well if so?  

Always had a bit of a "wow, so fast" moment when I've seen that elsewhere.

Reply:
@obmarg good point, thanks! Yes, I added micro- and nanoseconds now.

Reply:
Sweet - I did of course mean `us` there, but nanos is a good goal if we can acheive it

Reply:
@obmarg I added both. Micro (μs) and nano (ns). :-)